go/libkb/security_windows.go

```diff
@@ -0,0 +1,73 @@
+// copied from github.com/hectane/go-acl
+
```

Comment:
Is this enough attribution?

Reply:
It's a good point, you should check if this changes our licensing at all. Which license is this code under?